newm-server/src/main/kotlin/io/newm/server/features/distribution/eveara/EvearaDistributionRepositoryImpl.kt

```diff
@@ -1483,7 +1485,11 @@ class EvearaDistributionRepositoryImpl(
         user: User,
         collabs: List<Collaboration>
     ) {
-        require(collabs.all { it.status == CollaborationStatus.Accepted }) { "All Collaborations must be accepted!" }
+        require(
+            collabs.all {
```

Comment:
This is not quite right. I think we want to first `filter` collabs to only those where `royaltyRate > 0`. Then we want to do the check that all of those are `Accepted`.